src/studio/src/designer/backend/Controllers/DatamodelsController.cs

```diff
@@ -125,14 +125,32 @@ public async Task<IActionResult> UpdateDatamodel(string org, string app, string
         /// <param name="modelPath">The path to the file to be updated.</param>        
         [Authorize]
         [HttpPut]
-        [ProducesResponseType(201)]
+        [ProducesResponseType(204)]
```

Comment:
I've started using the StatusCodes class in the Microsoft.AspNetCore.Http namespace in places where I need the int value for a status code.

Reply:
Nice, much more readable. Thx.